### DISCUSSION TITLE
GitHub page not updating after a commit

### DISCUSSION THREAD

**Discussion by josephromano** [community/community#35232] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I just updated my index.md file for my GitHub page https://josephromano.github.io/PHYS1406/ and pushed that change to the repo.  But it doesn't look like the change was actually implemented.  Usually updates that I make to my pages show up after 2-3 minutes, but nothing has happened today and it has been over 6 hours since I pushed my change to the repo.  I know that other people have mentioned that this is sometimes related to cached files with the browser.  So I checked my pages using a different browser, but that didn't show that the update was implemented either.  Is there currently a problem with the GitHub pages server?

**Comment by airtower-luna** [comment, marked as answer]
I can see the latest deployment failed with a random-looking timeout error: https://github.com/josephromano/josephromano.github.io/actions/runs/3189173135/jobs/5202756231

There were two Actions "incidents" reported today:

* https://www.githubstatus.com/incidents/r5zfm12w4mbb
* https://www.githubstatus.com/incidents/smn1qtqvbsb6

Pages uses Actions to do the build and deployment, so I wouldn't be surprised if that's what caused the problem in your build. Both are listed as resolved now, so you might want to try again.

**Reply by airtower-luna** [reply]
I'm just noticing another potential issue (because it came up in #35197): Your site is *huge*. The [last successful build](https://github.com/josephromano/josephromano.github.io/actions/runs/3162573306) reports an artifact size of 1010 MB. The [documented limit is 1 GB](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits), so it's possible that the small changes since pushed your site over the limit.

**Reply by josephromano** [reply]
Thanks.  Do you know if I will be able to remove content doing a "git rm" command, or won't that go through now given that I exceed 1 GB?

**Reply by airtower-luna** [reply]
According to the documentation the only *hard* limit is the size of the deployed site, so `git rm` should be enough for that. Shrinking the repository history might still make sense if it's not going to cause trouble otherwise, to improve performance working with it. :sweat_smile:

**Reply by josephromano** [reply]
Thank you.  I'll try that.

**Reply by josephromano** [reply]
that worked!  thanks!

**Reply by airtower-luna** [reply]
Excellent! Marking the answer would be appreciated. :slightly_smiling_face:
